src/EnergyPlus/HVACControllers.cc

```diff
@@ -2309,18 +2309,18 @@ void TraceIterationStamp(EnergyPlusData &state,
     // Note that we do not go to the next line
     print(TraceFile,
           "{},{},{},{},{},{},{},{},{},{},{},{},",
-          General::LogicalToInteger(state.dataGlobal->ZoneSizingCalc),
-          General::LogicalToInteger(state.dataGlobal->SysSizingCalc),
+          state.dataGlobal->ZoneSizingCalc ? 1 : 0,
```

Comment:
`? :` produces actual code. You don't need to do that. `static_cast<int>` has the same effect and generates no code. `true` and `false` are already `1` and `0`, you only need to change the type.

Reply:
Yep, I knew there would be a better option, I just didn't take the time to verify that it would cast to zero and one.  Will make this change shortly.